light/example_test.go

```diff
@@ -66,7 +66,9 @@ func ExampleClient_Update() {
 		stdlog.Fatal(err)
 	}
 	defer func() {
```

Comment:
Maybe t.Cleanup?

Reply:
This does not have `t` present. I was thinking about this and double checked but usually examples do not take `t *testing.T`

Reply:
Ah, my bad